fedimintd/src/fedimintd.rs

```diff
@@ -182,9 +184,16 @@ impl Fedimintd {
 
     /// Attach default server modules to Fedimintd instance
     pub fn with_default_modules(self) -> Self {
-        self.with_module(LightningInit)
+        let s = self
+            .with_module(LightningInit)
             .with_module(MintInit)
-            .with_module(WalletInit)
+            .with_module(WalletInit);
+
+        if std::env::var_os(FM_USE_UNKNOWN_MODULE_ENV).is_some() {
+            s.with_module(UnknownInit)
+        } else {
+            s
+        }
```

Comment:
I'm not a fan of branching like this in production code just to make tests work. Aren't rust integration tests enough? If we really want devimint tests we can build a separate testing `fedimintd` with this extra module.

Reply:
Having separate binaries seem cumbersome for integrations and alike. There's really nothing hacky or insecure about this module, it's a proper module that... is not very useful in practice. If you don't like branching, I could just make `fedimintd` produce this module unconditionally. :D

I bet eventually default Fedimint will ship with some optional modules anyway, or even the current modules will become opitonal, and users will be choosing them at DKG. It's just we don't have this API surface yet, so an env var is used.